src/Scene/Scene.js

```diff
@@ -87,14 +90,26 @@ Scene.prototype.getPickPosition = function(mouse) {
     return this.gfxEngine.getPickingPositionFromDepth(mouse);
 };
 
+Scene.prototype.addStyle = function(style) {
```

Comment:
I'm not a fan of adding a ton of boiler-plate code, simply to forward method call.
Expose directly `stylesManager` instead.